Sdk/Track/Swrve.m

```diff
@@ -734,6 +809,10 @@ - (void)_deswizzlePushMethods
         SEL didFail = @selector(application:didFailToRegisterForRemoteNotificationsWithError:);
         [SwrveSwizzleHelper deswizzleMethod:didFail inClass:appDelegateClass originalImplementation:(IMP)didFailToRegisterForRemoteNotificationsWithErrorImpl];
         didFailToRegisterForRemoteNotificationsWithErrorImpl = NULL;
+        
+        SEL didReceive = @selector(application:didReceiveRemoteNotification:);
```

Comment:
Is it possible for variable name to be the same as the actual selector? Or perhaps same name with selector appended?